src/main.js

```diff
@@ -107,13 +107,12 @@ export class Draggable {
 
         this._pointerdown = (e) => {
             if (e.isPrimary && e.button === 0) {
-                bind(this._element, "pointermove", this._pointermove);
-                bind(this._element, "pointerup", this._pointerup);
-                bind(this._element, "contextmenu", preventDefault);
+                bind(document, "pointermove", this._pointermove);
+                bind(document, "pointerup", this._pointerup);
```

Comment:
The `pointermove` and `pointerup` events may fire for a different pointer when using multi-touch.
Maybe tying them to the same `pointerId` that triggered `pointerdown` can guard against that?